frontend/src/views/WheresMyGene/common/types.ts

```diff
@@ -1,7 +1,11 @@
 import { CSSProperties } from "react";
 
+export type Tissue = string;
+
 /** This object holds a cell type and its gene expressions. */
 export interface CellTypeSummary {
+  /** tissue name */
+  tissue: string;
```

Comment:
```suggestion
  tissue: Tissue;
```
Right?

Reply:
Oh thanks for catching this. I went ahead and update all types to use `Tissue` 🙏